helmfile/values/falco.yaml.gotmpl

```
@@ -49,14 +49,24 @@ customRules:
       condition: >
         (container.image.repository in (gcr.io/google_containers/hyperkube-amd64,
         gcr.io/google_containers/kube2sky, sysdig/agent, sysdig/falco,
-        sysdig/sysdig, falcosecurity/falco, quay.io/fluentd_elasticsearch/fluentd) or (k8s.ns.name = "kube-system"))
+        sysdig/sysdig, falcosecurity/falco, quay.io/fluentd_elasticsearch/fluentd),
```

Does this really work? Seems like there's one too many parentheses for me
```suggestion
        sysdig/sysdig, falcosecurity/falco, quay.io/fluentd_elasticsearch/fluentd,
```

---

I fixed it now , yes it works !!